appuio/stardog/values.yaml

```diff
@@ -92,7 +92,9 @@ metrics:
     repository: sscaling/jmx-prometheus-exporter
     tag: 0.12.0
     pullPolicy: IfNotPresent
-
+alerts:
```

Comment:
This will apply the "30m" value to all environments that use this helm chart. I don't think this is what we want?

This value should only be set in the repo which invokes this chart.

Reply:
That's true, done.